pkg/webhook/internal/metrics/metrics.go

```diff
@@ -32,9 +32,30 @@ var (
 		},
 		[]string{"webhook"},
 	)
+
+	// RequestTotal is a prometheus metric which is a counter of the total processed admission requests.
+	RequestTotal = func() *prometheus.CounterVec {
+		return prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "controller_runtime_webhook_requests_total",
+				Help: "Total number of admission requests by HTTP status code.",
+			},
+			[]string{"webhook", "code"},
```

Comment:
I think `path` would be the more sensible name than `webhook`.

Also, please use `const` for the label keys.

Reply:
> I think `path` would be the more sensible name than `webhook`.

This maybe a better name but I preferred to sick with the naming convention introduced by [`RequestLatency`](https://github.com/kubernetes-sigs/controller-runtime/blob/1fb86b6/pkg/webhook/internal/metrics/metrics.go#L33).
In my opinion have the labels consistent across metrics and avoiding a BC break was more important.

> 
> Also, please use `const` for the label keys.

Using `const` here could be done but would not fit within the current code based. See for example [pkg/webhook/internal/metrics/metrics.go](https://github.com/kubernetes-sigs/controller-runtime/blob/1fb86b6/pkg/webhook/internal/metrics/metrics.go#L33), [pkg/metrics/client_go_adapter.go](https://github.com/boekkooi-fresh/controller-runtime/blob/754026bd851020d5a94aa420e3a7613dbde76c65/pkg/metrics/client_go_adapter.go#L54), [pkg/metrics/workqueue.go](https://github.com/boekkooi-fresh/controller-runtime/blob/daa6f4d9e7c191732b38f310338bafc8e398ddc5/pkg/metrics/workqueue.go#L45) etc.
From my point of view I prefer to sick with the current code style.
Would you like me to open a separate PR to ensure const's are used for all metrics within `controller-runtime`?